oauthproxy.go

```diff
@@ -155,18 +155,6 @@ func setProxyUpstreamHostHeader(proxy *httputil.ReverseProxy, target *url.URL) {
 		director(req)
 		// use RequestURI so that we aren't unescaping encoded slashes in the request path
 		req.Host = target.Host
-		req.URL.Opaque = req.RequestURI
```

Comment:
This change was required because it was forcing the request uri to the original path. It seems you  no longer have to set opaque to retain escaped slashes, the test still passes with this change.